package-res/reportviewer/report.html

```diff
@@ -116,7 +116,10 @@
 
 <div id="reportArea" class="pentaho-transparent">
   <div id="reportPageOutline">
-    <iframe id="reportContent" frameborder="0"></iframe>
+    <form id="hiddenReportContentForm" method="post" target="reportContentIframe">
```

Comment:
Since we are already using jquery, is there a reason you are using a hidden form instead of using jquery to perform the post?
http://api.jquery.com/jquery.post/